Caster/ReflectionCaster.php

```diff
@@ -172,7 +172,11 @@ public static function castFunctionAbstract(\ReflectionFunctionAbstract $c, arra
 
         if (isset($a[$prefix.'returnType'])) {
             $v = $a[$prefix.'returnType'];
-            $v = $v->getName();
+            if (PHP_VERSION_ID >= 70000 && $v !== null) {
```

Comment:
this comparison in the original diff was because of the getType() method invocation which is only there in PHP 7 AFAIR.
So this outer `if` can be removed completely.